RecoLocalCalo/HGCalRecProducers/python/HGCalRecHit_cfi.py

```diff
@@ -232,3 +232,5 @@
 
 phase2_hgcalV9.toModify( HGCalRecHit , thicknessCorrection = [0.759,0.760,0.773] ) #120um, 200um, 300um
 phase2_hgcalV10.toModify( HGCalRecHit , thicknessCorrection = [0.781,0.775,0.769] ) #120um, 200um, 300um
+
+phase2_hfnose.toModify( HGCalRecHit , thicknessCorrection = [0.759,0.760,0.773])
```

Comment:
Is there a specific reason to use V9 factors instead of V10? Have you verified the validity of those numbers using the _calibration tool_ from @apsallid ?

Reply:
For the time being for the nose we are using the HFnoseV2(that was a clone of the HGCV9).
Here the validation of those scale correction
http://dalfonso.web.cern.ch/dalfonso/HGCAL/nosePlot_110_pre3/RecoFixedResponseGamma.png

After the 11_0 there will be a major update of the geometry that take into account the raddam/occupancy/layer separation and cost.
Those will require then both dEdX weights and scale corrections

Reply:
I added a dedicated thicknessNoseCorrection so that we can move independently the HGC and HFnose parameters